src/components/UTIL_PageMessages.component

```diff
@@ -15,7 +15,7 @@
         </style>
     </apex:outputText>
 
-    <div class="slds" id="page_messages">
```

Comment:
Yes. I went looking for some pages that used this component and where I could generate an error or message of some kind to confirm that it was working. Of course, you know the package better than I, so maybe there are some specific pages I should look at.

Also, it hadn't occurred to me that different pages would use different versions of SLDS. I has assumed that since all apex/vf is on the same api versions, the slds version would follow the same standard. Looking at the static resources though, there is a second slds zip file. I'll look into this further to make sure that the additional scope doesn't break pages that use the other static resource.